riakcs/tox.ini

```diff
@@ -2,18 +2,18 @@
 minversion = 2.0
 basepython = py27
 envlist =
-  {py27,py36}-unit
+  py{27,36}-unit
```

Comment:
Let's keep it as `py27,py36`, I am applying the same logic on all tox files in another PR and I choose the former wat just because there are less of the later

Reply:
Maybe could we change the other pr? I think this is more standard in other projects it seems